### PR TITLE
Issue#51

### PR DESCRIPTION
**Pull request by predatorprasad**
**created new yml file**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  my_service:
```
Comment (hwbrzzl) [inline docker-compose.yml:4]:
```suggestion
  goravel:
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    build:
+      context: .
+      dockerfile: Dockerfile
```
Comment (hwbrzzl) [inline docker-compose.yml:7]:
```suggestion
      ports:
        - "3000:3000"
      restart: always
```

**Review by hwbrzzl — COMMENTED** [review]
Hi, thanks for your PR! Just two tips, and please rebase this branch by `master`.

**Comment by predatorprasad** [comment]
Ok. am facing some issues I will open a new PR gimme a sec.

**Pull request closed without merging**